syntax = "proto3";

message RestartResponse {
    string uuid = 1;
}

message RestartRequest {
    string uuid = 1;
}

service Xray {
    rpc RestartXray(RestartRequest) returns (RestartResponse) {}
}
